apps/docs/src/pages/components.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Components
+description: All components showcase
+layout: ../layouts/MainLayout.astro
+---
+
+import Description from "../components/Description.astro";
+import Showcase from "../components/Showcase.astro";
+import ShowcaseCard from "../components/ShowcaseCard.astro";
+
+<Description>Below you can find all 12 components.</Description>
```

Comment:
I think we can skip the description for this one

Reply:
Done